ansible/roles/deploy-plugins/tasks/aws.yml

```diff
@@ -0,0 +1,44 @@
+- name: rename env_domain in preview_cdn.html for CDN
+  shell: |
+    echo "{{sunbird_portal_preview_cdn_url}}"
+    sed -i 's|cdn_url|{{sunbird_portal_preview_cdn_url}}|g' "{{currentws}}"/ansible/preview/preview_cdn.html
+  when: sunbird_portal_preview_cdn_url is defined
+  tags:
+    - preview
+    
+- name: delete batch
+  shell: |
+    aws s3 rm s3://{{ aws_s3_bucket }}/{{ folder_name }}/ --recursive
```

Comment:
in which task are we installing aws cli ?

Reply:
it has to be mannually installed on jenkins one time activity.